README.md

```diff
@@ -39,6 +39,47 @@ Or to some custom path:
 ```
 make PREFIX=/some/custom/path install
 ```
+## Ubuntu (and probably Debian)
+
+Install nvim, prerequisites, Rust + Cargo.
+
+* Use rustup.rs -- https://rustup.rs
+* This example has to be done as root since the rustup installer installs cargo in one's home dir
+
+Example (Tested on Ubuntu 23.04)
+
+``` shell
+#become root
+sudo su -
+
+#install prereq (see below)
+apt update
+apt install neovim
+apt install libgtk-4-dev -y
```

Comment:
This line (preceded with `sudo`) is the only line a Debian user should do.  Neovim can be installed multiple ways (Flatpak, AppImage, uncompressed tar on `~/.local/opt`, etc.).  It is perhaps a better idea to let the user choose their installation method.